firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -11,16 +11,18 @@ enum MovementStatus
 };
 
 // Current position tracking scale in eighth steps
+// P refers to positive, N refers to negative
+// Eighths refers to eight steps
 enum EighthStepsScale
 {
-  POS_EIGHTH_STEPS_PER_WHOLE_STEP = 8,
-  POS_EIGHTH_STEPS_PER_HALF_STEP = 4,
-  POS_EIGHTH_STEPS_PER_QUARTER_STEP = 2,
-  POS_EIGHTH_STEPS_PER_EIGHTH_STEP = 1,
-  NEG_EIGHTH_STEPS_PER_WHOLE_STEP = -8,
-  NEG_EIGHTH_STEPS_PER_HALF_STEP = -4,
-  NEG_EIGHTH_STEPS_PER_QUARTER_STEP = -2,
-  NEG_EIGHTH_STEPS_PER_EIGHTH_STEP = -1
+  P_EIGHTHS_PER_WHOLE_STEP = 8,
+  P_EIGHTHS_PER_HALF_STEP = 4,
+  P_EIGHTHS_PER_QUARTER_STEP = 2,
+  P_EIGHTHS_PER_EIGHTH_STEP = 1,
+  N_EIGHTHS_PER_WHOLE_STEP = -8,
+  N_EIGHTHS_PER_HALF_STEP = -4,
+  N_EIGHTHS_PER_QUARTER_STEP = -2,
+  N_EIGHTHS_PER_EIGHTH_STEP = -1
 };
```

Comment:
Since these are enums and not preprocessors, we should be able to just have `EIGHTS_PER_<size>_STEP` and do `-EIGHTS_PER_<size>_STEP`. Can you see if that works?

Reply:
The reason we did it this way was for runtime. This way is slightly better, with worse memory. Do you thin we should still change it?